services/connected_account_dev.yaml

```diff
@@ -37,6 +37,11 @@ Parameters:
     Description: |
       Unique ExternalId for Customer Organization; for cross-account Role Access and
       associating this template with a Customer Organization
+  Version:
```

Comment:
QUESTION: @flatherskevin, did you test that this works in both the create and the update path? In other words, did you test opening this template through `https://demo.cloudzero.com`? If not, let's test that together today.

Reply:
Erik showed me how to do that, but I would likely prefer walking through that flow with you as well for the full test so that this is fully tested. Since this is the initial entrypoint for a new company, if it immediately breaks for them, that will impact their confidence in our product